html/changelogs/lunanflag.yml

```diff
@@ -0,0 +1,7 @@
+author: Pirouette, StevIII
+
+delete-after: True
+
+changes:
+  - imageadd: "Added a Lunan banner, flag, and flagpatch to loadout."
+  - imageadd: "Replaced the Lunan accent sprite with one inspired by the flag."
```

Comment:
If the accent sprite is not being changed anymore, this line of the changelog have to go

Reply:
Fixed.